apps/frontend/src/auth/Logout.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect } from 'react';
+import { useAuth0 } from '@asap-hub/react-context';
+
+const Logout: React.FC<{}> = () => {
+  const { logout } = useAuth0();
+
+  useEffect(() => {
+    logout({ returnTo: globalThis.location.origin });
+  });
+
+  return 'Logging you out ...';
```

Comment:
Do we need a design for this?

Reply:
Not super urgently I guess, but for a 100% smooth experience yeah

Reply:
I'd be very interested to see the analytics for this, I always struggle to understand why logging out is such a common use case :sweat_smile: